requirements.txt

```diff
@@ -1,3 +1,2 @@
-blinker==1.4
-ts3==0.6.7
-wheel==0.24.0
+requests
+unittest
```

Comment:
You can run the code without unittest module. Read about requirements.txt separation for tests and for production.

Reply:
fixed